pycbc/inference/likelihood.py

```diff
@@ -123,11 +123,15 @@ def data(self):
         """Returns the data that was set."""
         return self._data
 
-    def loglikelihood(self, params):
+    def loglikelihood(self, *params):
         """This function should return the log likelihood of the given params.
         """
         raise ValueError("Likelihood function not set.")
 
+    def __call__(self, *params):
```

Comment:
Hey Collin can I recommend changing this to:
```
def __call__(self, params):
    self.loglikelihood(*params)
```

Samplers such as kombine or emcee will try to call the instance and pass it a list of the parameters.